hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/HoodieSparkCopyOnWriteTable.java

```diff
@@ -18,6 +18,8 @@
 
 package org.apache.hudi.table;
 
+import org.apache.avro.Schema;
```

Comment:
there is no other code change in this file. should avoid re-ordering imports here